vendor/github.com/ethereum/go-ethereum/accounts/abi/bind/base.go

```diff
@@ -120,19 +120,43 @@ func DeployContract(opts *TransactOpts, abi abi.ABI, bytecode []byte, backend Co
 	return c.address, tx, c, nil
 }
 
+//Adding retry logic here to deal with Infura's websocket instability.
```

Comment:
The commit message here should say livepeer/go-ethereum @ a8b8e87 but otherwise OK

Reply:
Yep changed.